recipes/android-ndk/all/test_package/conanfile.py

```diff
@@ -1,15 +1,28 @@
-from conans import ConanFile, tools
+import os
+from conans import ConanFile, tools, CMake
 
 
 class TestPackgeConan(ConanFile):
     settings = "os", "arch"
+    test_type = "build_requires"
+    generators = "cmake"
 
     def build(self):
-        pass
+        # It only makes sense to build a library, if the target os is Android
+        if self.settings.os == "Android":
+            cmake = CMake(self)
+            cmake.configure()
+            cmake.build()
 
     def test(self):
         if not tools.cross_building(self):
             if self.settings.os == "Windows":
                 self.run("ndk-build.cmd --version", run_environment=True)
             else:
                 self.run("ndk-build --version", run_environment=True)
+
+        # Run the project that was built using emsdk
```

Comment:
```suggestion
        # Run the project that was built using Android NDK
```

Reply:
😄  ... the `test_package` is mostly a copy of the one for `emsdk`